scanner.js

```diff
@@ -1473,8 +1475,13 @@ Scanner.prototype.revert_txids = function (callback) {
       var assets_transactions_bulk = self.AssetsTransactions.collection.initializeUnorderedBulkOp()
       assets_transactions_bulk.bulk_name = 'assets_transactions_bulk'
 
-      var txids = self.to_revert.slice(0, n_batch)
-      console.log('reverting txs (' + txids.length + ',' + self.to_revert.length + ')')
+      var txids = self.to_revert.slice(skip, Math.min(skip + limit, self.to_revert.length))
```

Comment:
No need to `Math.min(skip + limit, self.to_revert.length)` because `Array.prototype.slice` will do it any way.